tcms/core/utils/mailto.py

```diff
@@ -45,5 +45,5 @@ def mailto(  # pylint: disable=invalid-name
         kwargs={"fail_silently": False},
     )
     # This is to tell Python not to wait for the thread to return
-    email_thread.setDaemon(True)
+    email_thread.daemon = True
```

Comment:
It should be possible to specify `daemon=True` in the init metod for `Thread()` see https://docs.python.org/3.8/library/threading.html#threading.Thread